src/hardware/sblaster.cpp

```diff
@@ -294,6 +308,157 @@ static void InitializeSpeakerState()
 	}
 }
 
+static void configure_filters(Section_prop* config)
```

Comment:
Suggest setting config properties to `const Section_prop*`.

Reply:
Sure, I try to do that everywhere, but it takes constant vigilance. I'm used to immutable-by-default languages where mutation is the special case.